geonode/geoserver/helpers.py

```diff
@@ -2076,9 +2076,7 @@ def _render_thumbnail(req_body, width=240, height=200):
         im.thumbnail(
             (_default_thumb_size['width'], _default_thumb_size['height']),
             resample=Image.ANTIALIAS)
-        cover = resizeimage.resize_cover(
-            im,
-            [_default_thumb_size['width'], _default_thumb_size['height']])
+        cover = im.resize((_default_thumb_size['width'], _default_thumb_size['height']))
```

Comment:
`resize(...)` may not be the best choice here. Check the [documentation on what `resizeimage.resize_cover(...)` actually does](https://github.com/VingtCinq/python-resize-image#resize_coverimage-size-validatetrue-resampleimagelanczos) as it is not as simple as just resizing the image. Consider the [`ImageOps` module for PIL](https://pillow.readthedocs.io/en/stable/reference/ImageOps.html#PIL.ImageOps.fit) as an alternative.